src/main/java/qna/domain/AnswerRepository.java

```diff
@@ -5,7 +5,7 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 
 public interface AnswerRepository extends JpaRepository<Answer, Long> {
-    List<Answer> findByQuestion_idAndDeletedFalse(Long questionId);
+    List<Answer> findByQuestion_IdAndDeletedFalse(Long questionId);
```

Comment:
연관관계를 잘 맺어주셨으니, JPA 메서드 형식을 따르는게 어떨까요? :)
```suggestion
    List<Answer> findByQuestionIdAndDeletedFalse(Long questionId);
```